demo/AliceWantsAJsonCredential.md

```diff
@@ -78,7 +82,9 @@ Congradulations, you are now ready to start issuing JSON-LD credentials!
 - You have created a (non-public) DID for Faber to use to sign/issue the credentials - you will need to copy the DID that you created above into the examples below (as `issuer`).
 - You have created a (non-public) DID for Alice to use as her `credentialSubject.id` - this is required for Alice to sign the proof (the `credentialSubject.id` is not required, but then the provided presentation can't be verified).
 
-To issue a credential, use the `/issue-credential-2.0/send` (or `/issue-credential-2.0/create-offer`) endpoint, you can test with this example payload (just replace the "connection_id", "issuer" key, "credentialSubject.id" and "proofType" with appropriate values:
+To issue a credential, use the `/issue-credential-2.0/send` (or `/issue-credential-2.0/create-offer`) endpoint.  As mentioned above, if you are using the `/send` endpoint, you need to make sure to include `--no-auto` when starting both of the agents.
```

Comment:
Since the default option does not use `--no-auto`, then this phrasing is odd. It should say "use create-offer", unless you have used `--no-auto` vs. how you have worded it.

Not a big deal, but might as well fix it.

Reply:
Done